src/dk/ative/docjure/spreadsheet.clj

```diff
@@ -272,22 +272,32 @@
   (.createSheet workbook name))
 
 (defn create-workbook
-  "Create a new XLSX workbook with a single sheet and the data
-   specified. The data is given a vector of vectors, representing
+  "Create a new XLSX workbook. Data is a vector of vectors, representing
    the rows and the cells of the rows.
-
-   For example, to create a workbook with a sheet with
-   two rows of each three columns:
-
-   (create-workbook \"Sheet 1\"
-                    [[\"Name\" \"Quantity\" \"Price\"]
-                     [\"Foo Widget\" 2 42]])
-   "
-  [sheet-name data]
-  (let [workbook (XSSFWorkbook.)
-        sheet    (add-sheet! workbook sheet-name)]
-    (add-rows! sheet data)
-    workbook))
+  
+  Single sheet example: first argument is the sheet name, the second
+  argument is the sheet data.
+
+  (create-workbook \"SheetName1\" [[\"A1\" \"A2\"][\"B1\" \"B2\"]])
+
+  Multiple sheets example: one argument, a vector of alternating sheetnames and
+  associated sheetdata [SheetName1, SheetData1, SheetName2, SheetData2, ...]
+
+  (create-workbook [\"SheetName1\" [[\"FirstSheet A1\"][\"FirstSheet A2\"]]
+                    \"SheetName2\" [[\"SecondSheet A1\"][\"SecondSheet A2\"]]])"
+  ([sheet-name data]
+   (let [workbook (XSSFWorkbook.)
+         sheet    (add-sheet! workbook sheet-name)]
+     (add-rows! sheet data)
+     workbook))
+  
```

Comment:
I had some trouble following your definition of sheets, so I wrote an implementation of a cleaned up version (clean according to me, of course).

```clojure
([sheet-name data & name-data-pairs]
 ;; Incomplete pairs should not be allowed.
 {:pre [(even? (count name-data-pairs))]}
 ;; Use the two arity version of the function to make the initial workbook
 (let [workbook (create-workbook sheet-name data)]
   ;; Then iterate through the pairs adding sheets and rows
   (doseq [[s-name data] (partition 2 name-data-pairs)]
     (-> workbook
         (add-sheet! s-name)
         (add-rows! data)))
   workbook))
```


Single arity version
```clojure
([sheet-vec]
 ;; Incomplete pairs should not be allowed.
 {:pre [(even? (count sheet-vec))]}
 ;; Use the two arity version of the function to make the initial workbook
 (let [workbook (create-workbook (first sheet-vec) (second sheet-vec))]
   ;; Then iterate through the pairs adding sheets and rows
   (doseq [[s-name data] (partition 2 (drop 2 sheet-vec))]
     (-> workbook
         (add-sheet! s-name)
         (add-rows! data)))
   workbook))
```